unit_tests/test_events.py

```diff
@@ -66,8 +66,8 @@ class SctEventsTests(BaseEventsTest):  # pylint: disable=too-many-public-methods
     maxDiff = None
 
     def test_disruption_skipped_event(self):
-        with DisruptionEvent(nemesis_name="DeleteByRowsRange",
-                             node="target_node", publish_event=False) as nemesis_event:
+        with contextlib.suppress(UnsupportedNemesis), DisruptionEvent(
```

Comment:
```suggestion
        with pytest.raises(UnsupportedNemesis), DisruptionEvent(
```
since if we don't raise it's an error

Reply:
thanks, fixed